lib/ts/recipe/session/useSessionContext.ts

```diff
@@ -6,7 +6,7 @@ import { SessionContextType } from "./types";
 const useSessionContext = (): SessionContextType => {
     const ctx = React.useContext(SessionContext);
 
-    if (ctx.isDefault) {
+    if (ctx.isDefault === true) {
         throw new Error("Cannot use useSessionContext outside auth wrapper components.");
     }
```

Comment:
why this change? Cause not if isDefault is undefined, it will not throw the error - and from the types, it can be undefined. When will it be undefined?

Reply:
It's undefined if it's provided by `SessionAuth` (after the change). We only need that prop in the default/initialization value of the context. We could set it to false everywhere else but just omitting it is fine since we never actually want to set it to true.